### PR TITLE
CONTRIB: added gtest execution on PPC servers in Jenkins script

### PR DESCRIPTION
**Pull request by Di0gen**
Added GTEST execution on PPC servers.

### CODE REVIEW

**Comment by yosefe** [comment]
Before we merge it, need to make sure ucx passes on ppc

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1415/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/800/ for details (Mellanox internal link).

File: contrib/test_jenkins.sh [hunk 1]
```diff
@@ -58,6 +58,81 @@ echo "Build without IB verbs"
 ../contrib/configure-release --without-verbs
 make $make_opt
 
+if [ -n "$JENKINS_PPC_SERVER" ]; then
```
Comment (yosefe) [inline contrib/test_jenkins.sh]: please try to unite this with the other case (JENKINS_RUN_TESTS), avoid code duplication

Comment (Di0gen) [inline contrib/test_jenkins.sh]: 1st of all, I want stabilize the infra problems, benchmark and modules.
You will see false alarms with old code, and when all components of infra will be fixed, I'll fix code.

10x

File: contrib/test_jenkins.sh [hunk 2]
```diff
@@ -58,6 +58,81 @@ echo "Build without IB verbs"
...
+    make $make_opt clean
+
+    # todo: check in -devel mode as well
+    ../contrib/configure-release --with-mpi --prefix=$ucx_inst
+    make $make_opt install
+
+    ucx_inst_ptest=$ucx_inst/share/ucx/perftest
+
+    # hack for perftest, no way to override params used in batch
+    # todo: fix in perftest
```
Comment (shamisp) [inline contrib/test_jenkins.sh]: did u open a bug ?

**Comment by MattBBaker** [comment]
autobot build

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/816/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1427/ for details.

**Comment by Di0gen** [comment]
bot:retest

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1432/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/820/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1433/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/821/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1434/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/822/ for details (Mellanox internal link).

**Comment by miked-mellanox** [comment]
bot:retest

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1532/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/926/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
bot:retest

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1736/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/1132/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
bot:retest

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1905/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/1285/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
@Di0gen closing this due to inactivity

**Pull request closed without merging**